src/gapic-generator-typescript.ts

```diff
@@ -81,6 +83,7 @@ const mainServiceName = argv.mainService as string | undefined;
 const template = argv.template as string | undefined;
 const gapicValidatorOut = argv.gapicValidatorOut as string | undefined;
 const validation = (argv.validation as string | undefined) ?? 'true';
+const metadata = argv.metadata as string | undefined;
```

Comment:
Good catch! It was a string in the early version, then I converted it to boolean but missed it here.  Still works because boolean can be converted to string in JavaScript / TypeScript, but fixed for clarity.